blog-homepage/src/ArticleContent.js

```diff
@@ -0,0 +1,38 @@
+// ArticleContent /////////////////////////////////////////////////////////////
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function ArticleContent ({title, description, hasAudioAvailable, memberPreview}) {
+
+  // Determine whether to display the "🔊 Audio available" element.
+  const audioAvailableClasses =
+    hasAudioAvailable ? "audio-is-available" : "audio-is-available hidden";
+
+  // Determine whether to display the "★ Member preview" element.
+  const memberPreviewClasses =
+    memberPreview ? "member-preview" : "member-preview hidden";
+
+  return (
+      <div className="article-content">
+        <p className={audioAvailableClasses}>🔊 Audio available</p>
+        <p className={memberPreviewClasses}>★ Member preview</p>
+        <h3>
+          {title}
+        </h3>
+        <p className="article-description">
+          {description}
+        </p>
+      </div>
+  );
+
+}
+
+ArticleContent.propTypes = {
+  title: PropTypes.string.isRequired,
+  description: PropTypes.string,
```

Comment:
Nitpick: This could probably be required. I think one description is an empty string, but that's still a string.